openduck-py/openduck_py/routers/voice.py

```diff
@@ -248,8 +254,9 @@ def _inference(sentence: str):
                     "generated_tts",
                     audio=np.frombuffer(audio_chunk_bytes, dtype=np.int16),
                 )
-                for i in range(0, len(audio_chunk_bytes), 1024):
-                    await websocket.send_bytes(audio_chunk_bytes[i : i + 1024])
+                chunk_size = 8192 * 4
```

Comment:
this can be a const defined in the settings.py or something, CHUNK_SIZE = 8192*4

Reply:
good idea